easydita-knowledge-portal/functions.php

```diff
@@ -148,9 +148,9 @@ function easydita_knowledge_portal_get_hierarchy() {
 		$hierarchy = array();
 		if($post->ID) {
 			$hierarchy[] = $post->ID;
-			
```

Comment:
Again, be careful about what you stage and commit. You should use `git diff --staged` before committing each time to make sure you don't have any unnecessary changes (like these whitespace issues).